pkg/registry/types.go

```diff
@@ -204,6 +212,11 @@ type PackageProperty struct {
 	Version string `json:"version" yaml:"version"`
 }
 
+type DeprecatedProperty struct {
+	// Whether the bundle is deprecated
+	Deprecated bool `json:"deprecated" yaml:"deprecated"`
```

Comment:
How would the ambiguity be resolved when a bundle has two properties: one with deprecated=true and one with deprecated=false? I would suggest that the presence of a "deprecated" property in the property set indicates deprecation, and remove the bool value altogether.

Reply:
```golang
    type DeprecatedProperty string
    
    func (DeprecatedProperty) String() string {
        return "deprecated"
    }
````

@benluddy something like this? or maybe just a string constant?

Reply:
That's a great point - thanks for catching that

Reply:
@njhale I missed your comment and implemented the property as an empty struct - let me know your thoughts